SDWebImageSwiftUI/Classes/WebImage.swift

```diff
@@ -156,21 +158,33 @@ public struct WebImagePlayerView: View {
     /// True to start animation, false to stop animation.
     @Binding public var isAnimating: Bool
     
+    @State var show = false
     @ObservedObject var player: WebImagePlayer
     var placeholder: AnyView?
     
-    public init(player: WebImagePlayer, isAnimating: Binding<Bool> = .constant(false)) {
+    public init(player: WebImagePlayer, isAnimating: Binding<Bool> = .constant(false), isSupportDelayForShowingCachingImage: Bool = false, cachingID: String? = nil) {
         _isAnimating = isAnimating
         self.player = player
+        if let cachingID = cachingID, !player.cachingId.contains(cachingID) {
+            if isSupportDelayForShowingCachingImage && player.imageManager.image != nil, !player.isCacheImage {
+                player.imageManager.image = nil
+                player.imageManager.isFirstLoad = true
+                player.imageManager.isLoading = true
+            }
+            player.cachingId.insert(cachingID)
+        }
+        self.player.imageManager.isSupportDelayForShowingCachingImage = isSupportDelayForShowingCachingImage
```

Comment:
Since there is multiple `WebImagePlayerView` sharing one player, we should think about what makes sense interns of merging logic. 

For a boolean it probably makes sense that if any one `WebImagePlayerView` is requesting `isSupportDelayForShowingCachingImage` then the play should have `isSupportDelayForShowingCachingImage` 

i.e. for a boolean could be something like
```
self.player.imageManager.isSupportDelayForShowingCachingImage = isSupportDelayForShowingCachingImage || self.player.imageManager.isSupportDelayForShowingCachingImage
```

if its the time (CGFloat) then it could be something like
```
self.player.imageManager. minFakeLoadingDelay = max(minFakeLoadingDelay, self.player.imageManager.minFakeLoadingDelay)
```

Reply:
done